cmd/kube-controller-manager/app/storageversionmigrator.go

```diff
@@ -45,8 +46,8 @@ func startSVMController(
 	controllerContext ControllerContext,
 	controllerName string,
 ) (controller.Interface, bool, error) {
-	if !utilfeature.DefaultFeatureGate.Enabled(features.StorageVersionMigrator) ||
-		!clientgofeaturegate.InformerResourceVersion2.Enabled() {
+	if !oldutilfeature.DefaultFeatureGate.Enabled(features.StorageVersionMigrator) &&
+		!feature.Enabled(clientgofeaturegate.InformerResourceVersion3) {
```

Comment:
I think in my refactor this became !features.RecursiveReadOnlyMounts3.Enabled() . I guess the advantage in this refactor is that every featuregate has the env var assignment? How attached are you to the `feature.Enabled` ?

Reply:
I don't see why we wouldn't want ALL gates to have the same enablement options?

`feature.Enabled(` syntax was JUST to put the verb at the front of the LOC.  I find it easier to read as:

```
if !feature.Enabled(longpkgname.bigLongVariableName) {
```

than

```
if !longpkgname.bigLongVariableName.Enabled() {
```

That's REALLY the only argument :)